code/game/machinery/iv_drip.dm

```diff
@@ -33,6 +33,10 @@
 	var/obj/item/reagent_container
 	///Set false to block beaker use and instead use an internal reagent holder
 	var/use_internal_storage = FALSE
+	///If we're using the internal container, fill us UP with the below : list(/datum/reagent/water = 5000)
```

Comment:
```suggestion
	///If we're using the internal container, fill us UP with the below : list(/datum/reagent/medicine/salglu_solution = 5000)
```
I don't think we really need this reagent list in the comment here but if it is going to be then it should be correct at least :)

Reply:
It's a generic example so people instantly know the expected format :3

Reply:
I shouldn't do code reviews when I've just woken up :)